resource-dispatcher/documentation/deployment.md

```diff
@@ -1,6 +1,8 @@
 # Deployment
 
-This is fundamentally just a Python project that versions dependencies in `Pipenv` & `Pipenv.lock` files. Any system that can run projects using Pipenv should be able to run this application. If you are running this application standalone, you will need to set the environment variable `CONFIG_FILE` to the path of the config file to use. That said, there is a Helm chart available to get up and running quickly on OpenShift.
+This is fundamentally just a Python project that versions dependencies in `Pipenv` & `Pipenv.lock` files. Any system that can run projects using Pipenv should be able to run this application. If you are running this application standalone, you will need to set either the environment variable `CONFIG_FILE` to the path of the config file to use, or `CONFIG_DIR` to a directory of configuration YAML files to load.
```

Comment:
Not sure which README it should go, but would be good to include a bit more information on the directory option, e.g.: 
- does it only sources `.yml` and `.yaml` files found at the dir level - i.e.: no sub-directories
- the merge of the files is limited to `repositories` and `tasks`(?)
- the intended purpose of the merge - i.e.: does/does not merge values of an object(?)

Some of the same goes for the `deep_merge` functionality, so functionality relying on it could perhaps be mentioned in the same context as well

Reply:
I have significantly rethought the whole config merge strategy and written more documentation about it... Please see what you think now.

I'd like to somewhat avoid the user having to think _too_ hard about how the merge works while working on configs, so merge issues that will definitely cause a problem (like name or route collisions) are now specifically detected and warned against. Merge issues that will not cause a problem (like an identical repository dependency in two config files) are silently handled. Please let me know what you think of moving towards this more targeted config merge strategy (`deep_merge` is still around, just used for some more internal, runtime-level functions).